features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,33 @@
+Feature: when the current branch is deleted during a Git Town command, the initial previous branch remains the previous branch
```

Comment:
Feature: deleting the current branch maintains the existing previous branch setting